mathics/builtin/lists.py

```diff
@@ -562,10 +562,14 @@ class List(Builtin):
     summary_text = "specify a list explicitly"
     attributes = locked | protected
 
-    def apply(self, items, evaluation):
-        """List[items___]"""
-        items = items.get_sequence()
```

Comment:
@mmatera I would appreciate it if we would stop taking a variable of one type (some sort of Mathics expression and converting it into a variable of a different type (a list or tuple). 

In Basic or maybe some sort of interactive programming this kind of thing is done a bit. But in "production" code is is frowned upon, and in fact in typed programming languages it is disallowed.